src/core/tests/main.cpp

```diff
@@ -97,7 +97,7 @@ GTEST_API_ int main(int argc, char **argv)
     t.join();
     
     // exit without any destruction
-    dsn_terminate();
+    //dsn_terminate();
```

Comment:
I un-comment here because gcov would not work if run dsn_terminate() or killed by signal. OK, I revert it currently.